nominal/core/run.py

```diff
@@ -121,6 +121,12 @@ def _iter_list_attachments(self) -> Iterable[Attachment]:
     def list_attachments(self) -> Sequence[Attachment]:
         return list(self._iter_list_attachments())
 
+    def archive(self) -> None:
```

Comment:
should we also add unarchive?

Reply:
That's a great question except that an unarchive endpoint is not exposed